### DISCUSSION TITLE
Codemagic recent quality of service

### DISCUSSION THREAD

**Discussion by Reedyuk** [codemagic-ci-cd/codemagic-docs#2443] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Kotlin Multiplatorm

### Steps to reproduce

There has been alot of outages recently, it has caused the company i work for to miss out on a release.
Friday - 22nd March, Monday 25th and even today i can see we are limited to one virtual machine whereas we pay for 3.
Support have apologised, but really what i want is reliability. 
I like codemagic, but recently it has come into question, if we are to reconsider improving our redundancy for the cases where an outage happens (this defeats the point of using codemagic).

### Expected results

I would expect to have the number of virtual machines we pay for, available 24hrs - 7 days a week

### Actual results

Outages where builds fail for no reason, limitation of the number of virtual machines.

**Comment by icarusdust** [comment]
Hi @Reedyuk, I understand your frustration and let me once again apologize for the inconvenience caused. I can assure you that the team has already started working on the issue actively and currently, we are working together with our data center partner to resolve it as soon as possible, so your workflows can get back to normal as expected. If you want to follow up with the updates closely, check [this thread](https://github.com/orgs/codemagic-ci-cd/discussions/2424#discussioncomment-8930847) where our team keeps it up to date.
